acceptance/suites/tests/010-puppetserver-cli/subcommand/SERVER-297_common_behavior.rb

```diff
@@ -13,6 +15,18 @@
   assert(env['JARS_NO_REQUIRE'], "JARS_NO_REQUIRE missing")
 end
 
+step "ruby: Check that proxy env-variables are present"
+on(master, "#{proxy_env_vars} puppetserver ruby -rjson -e 'puts JSON.pretty_generate(ENV.to_hash)'") do
+  env = JSON.parse(stdout)
+  assert(env['HTTP_PROXY'], "HTTP_PROXY is missing")
+  assert(env['http_proxy'], "http_proxy is missing")
+  assert(env['HTTPS_PROXY'], "HTTPS_PROXY is missing")
+  assert(env['https_proxy'], "https_proxy is missing")
+  assert(env['NO_PROXY'], "NO_PROXY is missing")
+  assert(env['no_proxy'], "no_proxy is missing")
+  assert_nil(env['RANDOM_VARIABLE'], "RANDOM_VARIABLE is present and should not be")
```

Comment:
Not a big deal here, but not sure if it's worth even having the `RANDOM_VARIABLE` assertion here.